docs/guides/basic-rbac.mdx

```diff
@@ -155,7 +155,7 @@ export default function AdminDashboard() {
 ```
 
 <Callout type="info">
-  You can modify the behavior of the helper function to meet your needs. Maybe it will return the roles that the user has, or you could create a `protectByRole()` and have that function handle the redirect.
+  You can modify the behaviour of the helper function to meet your needs. Maybe it will return the roles that the user has, or you could create a `protectByRole()` and have that function handle the redirect.
```

Comment:
we use american spelling in our docs

```suggestion
  You can modify the behavior of the helper function to meet your needs. Maybe it will return the roles that the user has, or you could create a `protectByRole()` and have that function handle the redirect.
```